src/com/sun/ts/tests/internal/implementation/sjsas/jaxr/ee/javax_xml_registry/RegistryServicects/JAXRClient.java

```diff
@@ -34,8 +34,8 @@
 import java.rmi.*;
```

Comment:
There are no jakarta.xml.registry classes, I think that JAXRClient needs to be removed.

I did "ag --ignore .git  -l javax.xml.registry" locally to find the files that reference JAXR, which includes most of the files that you updated.  The JAXRClient is only used by the src/com/sun/ts/tests/jaxr/ files which I think you will delete.